db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -7040,7 +7078,8 @@ convertAddTimeIntervalToFunction(ValueNode base, TimeSpanNode timeSpanNode, bool
                 function = "ADD_YEARS";
```

Comment:
To echo Zhen's comment: We should probably support hours/minutes/seconds in this code path as well.
Alternatively, why not merge the two code paths? They seem redundant.